### PR TITLE
Add vouchallengers to policy

### PR DESCRIPTION
**Pull request by herniadlf**
Include vouchallengers definitions to Proof of Humanity policy. 

[Proposal](https://gov.proofofhumanity.id/t/phase-2-hip-58-removal-of-vouchallengers/2483?u=herniadlf)

### CODE REVIEW

File: policy.md [hunk 1]
```diff
@@ -101,6 +101,19 @@ The challenges types are the following:
 - Deceased: The submitter has existed but does not exist anymore.
   - The challenger can provide evidence that the submitter is dead such as a death certificate, an obituary, or public records.
   - The challenged submitter can provide a video of himself reading a recent block hash. Submitters not able to give recent proof of life are to be considered deceased.
+
+## Vouchallenges
+
+A vouchallenge is the act of using the vouch feature of registered humans to make the vouched profile vulnerable to a "Incorrect Submission" challenge. Once vouched, the challenge takes place (with the same voucher address or not) and the submitter loose the dispute. This practice is against the spirit of vouching. A voucher must check and ensure that the vouched profile is okay and not abuse from the submitter deposit. For that reason, it will be valid to ask for removal to those profiles that makes vouchallenges and the jurors will have to accept the removal.
```
Comment (bilinkis) [inline policy.md]: to ask for *the* removal *of* those profiles

**Review by bilinkis — CHANGES_REQUESTED** [review]
other than that comment, LGTM.

File: policy.md (hunk 1 above)
Comment (herniadlf) [reply policy.md]: fixed! Thx nico

**Comment by herniadlf** [comment]
Final version, no more changes are allowed.

**Comment by luiandresgonzalez** [comment]
Admission Criteria section needs to be updated as well.

**Comment by greenlucid** [comment]
This PR was outdated after https://github.com/Proof-Of-Humanity/poh-docs/pull/5

**Pull request closed without merging**